playbooks/roles/foreman_installer/tasks/packages.yml

```diff
@@ -1,10 +1,4 @@
 ---
-- name: 'Update packages'
-  yum:
-    name: '*'
-    update_cache: yes
-    state: latest
-
```

Comment:
So, in the Katello docs we tell users to ensure their systems are fully up to date before installing which makes this updating of packages sort of a "requirement" for installation. Thoughts?